src/main/java/com/arpnetworking/metrics/impl/BaseFileSink.java

```diff
@@ -414,5 +453,7 @@ protected void validate(final List<String> failures) {
         private static final Boolean DEFAULT_PRUDENT = Boolean.FALSE;
         private static final Boolean DEFAULT_DROP_WHEN_QUEUE_FULL = Boolean.FALSE;
         private static final Integer DEFAULT_MAX_QUEUE_SIZE = Integer.valueOf(500);
+        private static final String DEFAULT_MAX_FILE_SIZE = "100MB";
+        private static final String DEFAULT_TOTAL_SIZE_CAP = "20GB";
```

Comment:
I'm not sure that I agree with "only makes sense when the compression is set to false.". Can you elaborate?

Or was this cleared up during our conversation today?